projects/igniteui-angular/src/lib/core/styles/components/snackbar/_snackbar-theme.scss

```diff
@@ -94,39 +92,37 @@
         min-height: $snackbar-min-height;
         padding: $snackbar-padding;
         margin: rem(8px);
+        gap: rem(24px);
         color: var-get($theme, 'text-color');
         background: var-get($theme, 'background');
         backface-visibility: hidden;
         box-shadow: var-get($theme, 'shadow');
         border-radius: var-get($theme, 'border-radius');
         backdrop-filter: blur(8px);
-    }
-
-    %igx-snackbar-message {
-        @include animation('fade-in' .35s ease-out);
-    }
 
-    %igx-snackbar-button {
-        background: transparent;
-        color: var-get($theme, 'button-color');
-        border: 0;
-        line-height: $snackbar-button-line-height;
-        margin-inline-start: $snackbar-button-left-margin;
-        text-transform: uppercase;
-        user-select: none;
-        font-weight: $snackbar-button-font-weight;
-        -webkit-tap-highlight-color: transparent;
-        outline: none;
-        transition: color .2s ease;
-        font-size: inherit;
-        font-family: inherit;
-        cursor: pointer;
-        @include animation('fade-in' .35s ease-out);
-
-        &:hover {
+        [igxButton] {
```

Comment:
@simeonoff You will review the actual styles but are all of the overrides included here actually needed?

Reply:
We are checking. Will compare to the UI kit and web components framework and remove any unnecessary styles.